main.go

```diff
@@ -107,6 +134,19 @@ func StartProcessors(ctx context.Context, cfg *config.Configuration, errChan cha
 	return processors, nil
 }
 
+func waitForStartup(ctx context.Context, errChan chan processor.Error) error {
+	startupTimer, cancel := context.WithTimeout(ctx, 5*time.Second)
```

Comment:
Hard coded. Probably OK but we've tended to avoid hard coding, often. Could use a constant at least?